src/Features/Core/Portable/MakeMethodAsynchronous/AbstractMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -47,31 +49,31 @@ internal abstract class AbstractMakeMethodAsynchronousCodeFixProvider : CodeFixP
             }
 
             var symbol = semanticModel.GetDeclaredSymbol(node, cancellationToken) as IMethodSymbol;
+            var isEntryPoint = symbol != null && symbol.IsStatic && IsLikelyEntryPointName(symbol.Name, context.Document);
```

Comment:
very minor nit: since this is a heuristic, it might be worthwhile to mention taht it's not totally accurate, but we're ok with the unlikely false positive for this feature.